infrastructure/datastore/latest_entry_repository.go

```diff
@@ -0,0 +1,74 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/mjibson/goon"
+	"github.com/pkg/errors"
+	"github.com/utahta/momoclo-channel/domain"
+	"github.com/utahta/momoclo-channel/domain/entity"
+	"google.golang.org/appengine/datastore"
+)
+
+type LatestEntryRepository struct{}
+
+func NewLatestEntryRepository() *LatestEntryRepository {
+	return &LatestEntryRepository{}
+}
+
+func (repo *LatestEntryRepository) Save(c context.Context, l *entity.LatestEntry) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method LatestEntryRepository.Save should have comment or be unexported